### PR TITLE
Etcd quotaapi.Server DeleteConfig implementation

### PR DESCRIPTION
**Pull request by codingllama**

### CODE REVIEW

**Comment by codingllama** [comment]
Issue #490

**Comment by codingllama** [comment]
TODO hunting season to begin soon. :)

File: quota/etcd/quotaapi/quota_server_test.go [hunk 1]
```diff
@@ -492,16 +492,102 @@ func runUpsertTest(ctx context.Context, test upsertTest, rpc upsertRPC, rpcName
...
+			if _, err := quotaClient.CreateConfig(ctx, &quotapb.CreateConfigRequest{
+				Name:   cfg.Name,
+				Config: cfg,
+			}); err != nil {
+				t.Fatalf("%v: CreateConfig(%q) returned err = %v", test.desc, cfg.Name, err)
+			}
+		}
+
+		if _, err := quotaClient.DeleteConfig(ctx, test.req); err != nil {
+			t.Errorf("%v: DeleteConfig() returned err = %v", test.desc, err)
```
Comment (daviddrysdale) [inline quota/etcd/quotaapi/quota_server_test.go]: maybe include `test.req.Name` here?

Comment (codingllama) [inline quota/etcd/quotaapi/quota_server_test.go]: Done.

**Pull request merged**